devtools-html/debugger.html
# Ensure SearchInput retains value when open and clears when closed

In looking at #5366, I noticed a problem:

* Command+F opens the search and focuses on the `input`, but then clicking into the editor and Command+F'ing again does not.  I would expect the input to be focused and its text selected.

### Summary of Changes

* Clearing the state: query is important 
* Retaining the value when Command+F is pressed is important

### Test Plan

* Open a source in the editor
* Press Command+F, type a keyword
* Click into the editor, press Command+F again, ensure the search text is still there and selected
* Press Escape, the search box should close
* Press Command+F again, the input should display but be clear of text ("")

### Feedback Wanted

* `this.$input.$input.value` seems gross; what's a better way of having these two components interact more cleanly?
